harness/determined/common/experimental/_util.py

```diff
@@ -0,0 +1,18 @@
+import enum
+
+from determined.common.api import bindings
+
+
+class OrderBy(enum.Enum):
+    """
+    Specifies whether a sorted list of objects should be in ascending or
+    descending order.
+    """
+
+    ASCENDING = bindings.v1OrderBy.ASC.value
+    ASC = bindings.v1OrderBy.ASC.value
+    DESCENDING = bindings.v1OrderBy.DESC.value
+    DESC = bindings.v1OrderBy.DESC.value
+
+    def _to_bindings(self) -> bindings.v1OrderBy:
```

Comment:
Also a `_from_bindings`? Is that dumb?

Reply:
thought about this at some point, but couldn't come up with a use case where you'd need to get `OrderBy` back from the REST API. don't want to preemptively add something that will just be dead code. we can add it if it's needed at some point. 🤷

Reply:
I admire that conservatism. I think you've made the right choice.